MessageCenter/Classes/Views/Chatting/GroupChannelChattingViewController.swift

```diff
@@ -116,18 +116,16 @@ class GroupChannelChattingViewController: UIViewController, SBDConnectionDelegat
             self.cachedMessage = true
         }
         
-//        if SBDMain.getConnectState() == .closed {
-//            ConnectionManager.login { (user, error) in
-//                guard error == nil else {
-//                    return
-//                }
-//            }
-//        }
-//        else {
+        if SBDMain.getConnectState() == .closed {
+            
+            // need to connect again here
```

Comment:
Mark important notes and todos as TODO.
```suggestion
            // TODO: need to connect again here
```